lib/vdsm/storage/nbd.py

```diff
@@ -119,15 +119,28 @@ def start_server(server_id, config):
     if vol.isShared() and not cfg.readonly:
         raise se.SharedVolumeNonWritable(vol)
 
-    if cfg.bitmap and vol.getFormat() != sc.COW_FORMAT:
-        raise se.UnsupportedOperation("Cannot export bitmap from RAW volume")
+    if cfg.bitmap:
+        if vol.getFormat() != sc.COW_FORMAT:
+            raise se.UnsupportedOperation(
+                "Cannot export bitmap from RAW volume")
+
+        # The bitmap must exist in the volume, and may exist in the
+        # backing chain.
+        bitmap_chain = _find_bitmap(vol.volumePath, cfg.bitmap)
+        if not bitmap_chain:
+            raise se.BitmapDoesNotExist(
+                reason=f"Bitmap does not exist in {vol.volumePath}",
+                bitmap=cfg.bitmap)
 
     _create_rundir()
 
-    using_overlay = cfg.bitmap and vol.getParent() != sc.BLANK_UUID
+    # If the bitmap exists in the volume backing chain, we need to
+    # create an overlay exporting all bitmap from the entire chain.
```

Comment:
`bitmap` --> `bitmaps` (plural).

Reply:
We are exporting a single bitmap - it may exist in multiple images in the chain, so
we need to merge all the copies from the chain into the overlay.

Reply:
I replaced "all" with "the". If you want to understand this better look in _create_overlay and the relevant tests.